### ISSUE TITLE
Mastodon: Support custom emojis in display names

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#7486]

**Comment by MrPetovan** [comment]
I found only one place where we have access to custom emojis, in `Protocol\ActivityPub\Processor`. Are emojis also present in the AP response for contact probing?

**Comment by MrPetovan** [comment]
Paging @annando

**Comment by annando** [comment]
I don't think so. There is some API endpoint, I think. Hadn't you used this for the custom emoji addon?

**Comment by MrPetovan** [comment]
Yes but I was wondering if there was a better way than needing another database table just for this.

**Comment by AlfredSK** [comment]
It's not only Mastodon. Pleroma has that feature too.

![customemojisinnames2](https://user-images.githubusercontent.com/3656054/68540727-0f4ef800-0397-11ea-8615-e054d36ff0b5.png)

![customemojisinnames](https://user-images.githubusercontent.com/3656054/68540729-17a73300-0397-11ea-92e9-0f6ea7a80b09.png)

In this case the bunnies are hopping a.k.a. animated. :-)


```<div title=":abunhdhappyhop: :abunhdhappy: :abunhdhop: :abunhd: :abunhdhappyhop: :abunhdhappy:" class="user-name"><img src="https://pleroma.site/emoji/bunhd/abunhdhappyhop.png" alt="abunhdhappyhop" title="abunhdhappyhop" class="emoji"> <img src="https://pleroma.site/emoji/bunhd/abunhdhappy.png" alt="abunhdhappy" title="abunhdhappy" class="emoji"> <img src="https://pleroma.site/emoji/bunhd/abunhdhop.png" alt="abunhdhop" title="abunhdhop" class="emoji"> <img src="https://pleroma.site/emoji/bunhd/abunhd.png" alt="abunhd" title="abunhd" class="emoji"> <img src="https://pleroma.site/emoji/bunhd/abunhdhappyhop.png" alt="abunhdhappyhop" title="abunhdhappyhop" class="emoji"> <img src="https://pleroma.site/emoji/bunhd/abunhdhappy.png" alt="abunhdhappy" title="abunhdhappy" class="emoji"></div>```

**Comment by MrPetovan** [comment]
I've decided that the cost of this feature (in work and database space afterwards) isn't worth it.

**Comment by nightred** [comment]
This is becoming far more common in all instances, it does make post titles and names look odd.
